doc/vimteractive.txt

```diff
@@ -104,15 +109,26 @@ ALT-S sends all lines from the start to the current line.
 ==============================================================================
 3. Extending functionality                            *vimteractive-extending*
 
+To add a new interpreter to Vimteractive, you should define
+g:vimteractive_commands variable. For example:
+
+	let g:vimteractive_commands = { 'pythonasync': 'python -m asyncio' }
```

Comment:
This is cool. I like the idea of an asynchronous REPL.

Reply:
Unfortunately, it would be there only in 3.8.